CODEOWNERS

```
@@ -0,0 +1,8 @@
+# Below are suggested reviewers for PRs to packages in this repository.
+#
+# These are suggestions; it is fine to have your changes reviewed by someone
+# else.
+
+/pkgs/appengine/               @jonasfj @sigurdm
+/pkgs/gcloud/                  @jonasfj @sigurdm
```

```suggestion
/pkgs/appengine/               @dart-lang/dart-pub-dev-team
/pkgs/gcloud/                  @dart-lang/dart-pub-dev-team
```

Though I wish we could rename the team `dart-pub-team`, but maybe we'll get there :rofl: I'm investigating outside this PR.

---

`dart-pub-team` sounds like a good idea

---

It looks like for a team to be used here it must have explicit write access to the repo (not just that all the members have write access).